apps/src/code-studio/showProjectAdmin.js

```diff
@@ -83,14 +83,16 @@ export default () => {
   }
 
   $('#disable-auto-moderation').click(function () {
-    alert("You clicked the disable automated image moderation button. It doesn't do anything yet because our moderation filter is turned off, but it will be functional soon. Thanks for your patience with this work-in-progress button!");
+    dashboard.project.disableAutoContentModeration();
+    alert("Our moderation filter is currently turned off for all projects, but your choice will be saved and applied when we re-enable the moderation service.");
     $('#disable-auto-moderation').hide();
     $('#moderation-explanation').hide();
     $('#enable-auto-moderation').show();
```

Comment:
It would be nice if we didn't switch the displayed buttons until/unless the request to enable/disable moderation succeeds.   Also a great opportunity to use async/await!

```js
// in project.js
// Make these methods return a promise that resolves when the request succeeds, or rejects if it fails.
disableAutoContentModeration() {
  return new Promise((resolve, reject) => {
    channels.update(`${this.getCurrentId()}/disable-content-moderation`, (err) => {
      err ? reject(err) : resolve()
    });
  });
},

// in showProjectAdmin.js
// Make the click handler async, and await disableAutoContentModeration before changing the buttons.
$('#disable-auto-moderation').click(async function() {
  await dashboard.project.disableAutoContentModeration();
  alert("Our moderation filter...");
  $('#disable-auto-moderation').hide();
  $('#moderation-explanation').hide();
  $('#enable-auto-moderation').show();
});
```